src/Components/Confirmation/ConfirmationSteps.tsx

```diff
@@ -127,7 +127,13 @@ function Expenses() {
 
   const allExpenses = () => {
     if (formData.expenses.length === 0) {
-      return <FormattedMessage id="confirmation.none" defaultMessage="None" />;
+      return (
+        <ConfirmationItem
+          label=""
```

Comment:
@IrynaKolh Small suggestion, it may be clearer to make the label param optional instead of passing an empty string here.